app/components/pages/Onboarding.tsx

```diff
@@ -0,0 +1,166 @@
+import { Component } from 'vue-property-decorator';
+import { Onboarding } from 'streamlabs-beaker';
+import TsxComponent from 'components/tsx-component';
+import { OnboardingService } from 'services/onboarding';
+import { Inject } from 'services/core/injector';
+import Connect from './onboarding-steps/Connect';
+import ObsImport from './onboarding-steps/ObsImport';
+import StreamlabsFeatures from './onboarding-steps/StreamlabsFeatures';
+import Optimize from './onboarding-steps/Optimize';
+import FacebookPageCreation from './onboarding-steps/FacebookPageCreation';
+import ThemeSelector from './onboarding-steps/ThemeSelector';
+import { IncrementalRolloutService, EAvailableFeatures } from 'services/incremental-rollout';
+import { UserService } from 'services/user';
+import { $t } from 'services/i18n';
+import styles from './Onboarding.m.less';
+import { throws } from 'assert';
+
+@Component({})
+export default class OnboardingPage extends TsxComponent<{}> {
+  @Inject() onboardingService: OnboardingService;
+  @Inject() incrementalRolloutService: IncrementalRolloutService;
+  @Inject() userService: UserService;
+
+  currentStep = 1;
+  importedFromObs = false;
+  processing = false;
+  fbSetupEnabled: boolean = null;
+
+  stepsState = [{ complete: false }, { complete: false }, { complete: false }];
+
+  checkFbPageEnabled() {
+    if (this.fbSetupEnabled === null || !this.userService.isLoggedIn()) return;
```

Comment:
Maybe I'm reading this wrong, but I don't see how this can ever become not-null if this function returns here before fetching the value.

Reply:
yea thats supposed to be !== null good catch